README.md

```diff
@@ -374,13 +376,43 @@ let app =
         ]
 ```
 
+### subRoute
+
+`subRoute` checks the current path begins with the `path` string, all `route*` and `subRoute*` handlers within the `handler` parameter will now be rooted to, and relative with respect to this path.
+
```

Comment:
Not sure I follow you when you say "will partially apply the route", there is no function partial application? Unless my typo (see other comment) mislead you.